apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -197,21 +199,27 @@ export default class AmazonFutureEngineerEligibilityForm extends React.Component
     return missingRequiredFields;
   }
 
+  // Collapse dropdown if 'Escape' is pressed
+  onKeyDown(e) {
+    if (e.keyCode === 27) {
```

Comment:
Where did you find that Escape is 27? (mostly looks for documentation as I've never seen `keyCode` before!)

Reply:
I stole this logic from https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/CheckboxDropdown.jsx#L32! So I guess I was trusting that whoever wrote this researched it. But I'll double check that the escape key works!

Reply:
Should've commented here- this works:)

Reply:
thanks for checking!